internal/runners/export/deptree/artifacts.go

```diff
@@ -0,0 +1,142 @@
+package deptree
```

Comment:
Please note the code in runners/export/deptree was writen for debugging purposes. I made architectural shortcuts for sure, this code is decidedly unstable and does not follow best practice. I've committed it as-is because it still holds a lot of value to us, but please take it for what it is in your review.